extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -166,6 +166,7 @@ public QuartzScheduler(SchedulerContext context, QuartzSupport quartzSupport, Sc
                         String cron = SchedulerUtils.lookUpPropertyValue(scheduled.cron());
                         if (!cron.isEmpty()) {
                             if (SchedulerUtils.isOff(cron)) {
+                                this.pause(identity);
```

Comment:
Let's pause the job disabled job to effectively let Quartz know that this should not be invoked in any way. 
When the job is enabled/turned on, it'll be rescheduled and executed again.

Reply:
This results in a `org.quartz.Scheduler.pauseJob(JobKey)` invocation, right? What happens if the job does not exist yet? Is it a no-op?

Reply:
If the job does not exists, it is a no-op.
I'll add a test case in the integration test project to make sure we won't have a regression here.